v2/java/instructions.md

```diff
@@ -0,0 +1,39 @@
+# AdSense Management API v2 Java Examples
+
+## Instructions for the AdSense Management API v2 Command-Line Sample
+
+### Browse Online
+
+  - [Browse Source](http://code.google.com/p/google-api-java-client/source/browse?repo=samples#hg/adsense-cmdline-sample)
+  - Main file to run all examples: [AdSenseSample.java](http://code.google.com/p/google-api-java-client/source/browse/adsense-cmdline-sample/src/main/java/com/google/api/services/samples/adsense/cmdline/AdSenseSample.java?repo=samples)
+
+### Register Your Application
+
+  - Visit the [Google Cloud console](https://cloud.google.com/console/start/api?id=adsense)
+  - If necessary, sign in to your Google Account, select or create a project,
+  and agree to the terms of service.  Click Continue.
+  - Select "Installed application" and choose type "Other" under the Installed
```

Comment:
I believe this workflow has changed, however, it may be different for AdSense Management API. See AdMob API instructions for using new OAuth Client ID credentials: https://github.com/googleads/googleads-admob-api-samples/tree/master/java/v1

Reply:
Nice catch! I updated the instructions and moved this to README.md while I was at it.